packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-color/example.html

```diff
@@ -66,6 +66,20 @@ <h2>Examples</h2>
       <span slot="help">This is a help message</span>
     </gux-form-field-color>
   </fieldset>
+
+  <fieldset>
+    <legend>Tooltip</legend>
+
+    <gux-form-field-color>
+      <input slot="input" type="color" name="e-1" value="#cc3ebe" />
+      <label slot="label">Default</label>
+      <gux-form-field-tooltip
```

Comment:
Only in gux-form-field-color for now, will update other form-fields once this is ok.